spec/passaporte_web/identity_spec.rb

```diff
@@ -245,10 +245,10 @@
   describe "#authenticate", vcr: true do
     let(:identity) { described_class.find("5e32f927-c4ab-404e-a91c-b2abc05afb56") }
     it "should return true if the password is correct" do
-      identity.authenticate('123456').should be_true
+      expect(identity.authenticate('123456')).to be_truthy
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.